pai-management/k8sPaiLibrary/template/kubelet.sh.template

```diff
@@ -46,4 +46,5 @@ docker run \
   --logtostderr=true \
   --image-gc-high-threshold=98 \
   --image-gc-low-threshold=95 \
+  --pod-infra-container-image {{ clusterconfig['dockerregistry'] }}/pause-amd64:3.0 \
```

Comment:
👍 
China network can't get the image 'gcr.io/google_containers/pause-amd64'.
This solved the problem, we can host the image on dockerhub 'docker.io/openpai/pause-amd64'.

Reply:
Did you test pulling these images from docker.io/openpai? And how about other images, such as etcd, apiserver, etc.?

Reply:
yes,
All tested.

Reply:
Yes, I could install k8s cluster on ustc cluster using the docker.io/openpai.

Reply:
ummm...
i writed
docker-registry: docker.io/openpai
and i got
2019-05-20 09:58:40,475 [INFO] - deployment.k8sPaiLibrary.maintainlib.kubectl_install : Wait 5s, and retry it later.
The connection to the server 192.168.9.142:8080 was refused - did you specify the right host or port?
2019-05-20 09:58:45,582 [WARNING] - deployment.k8sPaiLibrary.maintainlib.common : There will be a delay after installing, please wait.
2019-05-20 09:58:45,583 [ERROR] - deployment.k8sPaiLibrary.maintainlib.kubectl_install : kubectl ready test failed. Exit paictl.